.goreleaser.yaml

```diff
@@ -0,0 +1,89 @@
+# For more information on configuring GoReleaser, refer to the documentation:
+# https://goreleaser.com
+#
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+version: 2
+
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
```

Comment:
It seems odd to have `go generate` called at build time. I suspect this was from a template that assumes generated content is not checked into the repo?

Reply:
Correct, this was just included from the example template they had.